internal/service/util.go

```diff
@@ -174,7 +175,7 @@ func serveURLBase(r *http.Request) string {
 		ps = fp[0]
 	}
 
-	return fmt.Sprintf("%v://%v/", ps, ph)
+	return fmt.Sprintf("%v://%v%v", ps, ph, strings.TrimRight(conf.Configuration.Server.BasePath, "/"))
```

Comment:
@nfickas I think this is a bug.  It shows up for me when using the UI at `localhost:9000`.  The links genned for the UI are missing the `/` after the host.  That's because it's getting trimmed off here.

The fix is to trim the `/` in case it is there in the configuration, and then add it explicitly:
```
	path := strings.TrimRight(conf.Configuration.Server.BasePath, "/")

	return fmt.Sprintf("%v://%v%v", ps, ph, path+"/")
```
If you're able to fix this that's great, or let me know.

Reply:
Actually a better approach is to sanitize the base path as soon as the config is read.  Then it is certain to be accurate everywhere, and doesn't cost cycles on every request.

Reply:
Fixed by https://github.com/CrunchyData/pg_featureserv/commit/910ca4dd720500362d2d36c927a2b0771007c478.

Although this could be done better as noted above.